openapi.yaml

```diff
@@ -1437,11 +1441,109 @@ components:
       properties:
         PaymentType:
           type: string
-          description: Defines if the transfer is authorized for a single transaction or reoccurring
+          description: TODO: Defines if the transfer is authorized for a single transaction or reoccurring
```

Comment:
Should this be merged in or broken out as an issue?

Reply:
Paygate doesn't support WEB files yet, that's a stub from https://github.com/moov-io/paygate/issues/16